src/base.jl

```diff
@@ -2180,9 +2180,23 @@ function check(sys::System)
     critical_components_check(sys)
     adequacy_check(sys)
     check_subsystems(sys)
+    check_base_powers(sys)
     return
 end
 
+"""
+Check that base power on static injection components are nonzero.
+"""
+function check_base_powers(sys::System)
```

Comment:
We don't want to modify the systems in runchecks. As the name implies this is meant to do the checks. You can keep the warning if the base power it below 1.0 that is looks like a bad data input. But don't modify the system